cmd/application.go

```diff
@@ -78,12 +80,27 @@ var applicationDeleteCmd = &cobra.Command{
 		return nil
 	},
 	Run: func(cmd *cobra.Command, args []string) {
-		err := application.Delete(args[0])
-		if err != nil {
-			fmt.Println(err)
-			os.Exit(1)
+		appName := args[0]
+		var confirmDeletion string
+
+		if applicationForceDeleteFlag {
+			confirmDeletion = "y"
+		} else {
+			fmt.Printf("Are you sure you want to delete the application: %v? [y/N] ", appName)
+			fmt.Scanln(&confirmDeletion)
+		}
+
+		if strings.ToLower(confirmDeletion) == "y" {
+			err := application.Delete(appName)
+			if err != nil {
+				fmt.Println(err)
+				os.Exit(1)
+			}
+			fmt.Printf("Deleted application: %s\n", args[0])
+		} else {
+			fmt.Printf("Aborting deletion of application: %v\n", appName)
+			os.Exit(0)
```

Comment:
don't use exit here if you are not returning error code, just `return`. Or in this case, there doesn't have to be anything

Reply:
fixed